techmatters/flex-plugins
# [WIP] Upgrade dependencies version

## Description
This PR upgrades the depencies of `flex-plugins/plugin-hrm-form`.

`npm` still complains about some vunerabilities, but none of them are critical. Trying to `npm audit fix` breaks the app because of breaking changes.


### Related Issues
Fixes https://bugs.benetech.org/browse/CHI-1028

### Verification steps
- fetch code
- remove node_modules/
- run `NBUNDLED_REACT=true npm ci`
- Check the Aselo still works fine
